contracts/BtcLightClient.template

```diff
@@ -289,7 +289,7 @@ contract BtcLightClient is ILightClient, System, IParamSubscriber{
     } else if (2*maxWeight < count && count <= (2*maxWeight + 3*maxWeight/4)) {
       return 3*maxWeight - count;
     } else {
-      return count/4;
+      return count/4 - (7*maxWeight)/16;
```

Comment:
@jackcrypto9527 this change will remove the jump while keeping the slope. guess it's as good as anything

Reply:
Could be, since this is more of a PM discussion lets revert and re-open later on